### PR TITLE
Create ui-c8y-1019-7-9-graft-release-cd-fix-nightly-failure-in-device-protocol-sigfox-5751.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dafk

Ticket: [DM-3402](https://cumulocity.atlassian.net/browse/DM-3402)
Original PR: [5751](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5751)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-7-9-graft-release-cd-fix-nightly-failure-in-device-protocol-sigfox-5751.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: [GRAFT][release/cd] Fix nightly failure in deviceProtocolSigfox (#5751)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-7-9-graft-release-cd-fix-nightly-failure-in-device-protocol-sigfox-5751.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix Sigfox device protocol nightly build failure
```

File: content/change-logs/ui-c8y-1019-7-9-graft-release-cd-fix-nightly-failure-in-device-protocol-sigfox-5751.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3402
+version: 1019.7.9
+---
+[GRAFT][release/cd] Fix nightly failure in deviceProtocolSigfox (#5751)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-7-9-graft-release-cd-fix-nightly-failure-in-device-protocol-sigfox-5751.md:17]: The Claude AI suggested to use the following body:
```suggestion
Resolved an issue causing nightly build failures in the Sigfox device protocol code. This fixes the build breakage to ensure smooth nightly builds.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by janhommes** [comment]
fix for test.

**Pull request closed without merging**